x-pack/plugins/canvas/canvas_plugin_src/renderers/shape/index.js

```diff
@@ -41,22 +41,20 @@ export const shape = () => ({
 
       // adjust viewBox based on border width
       let [minX, minY, shapeWidth, shapeHeight] = initialViewBox;
-      const borderOffset = strokeWidth / 2;
+      const borderOffset = strokeWidth;
 
       if (width) {
-        const xScale = (shapeWidth - minX) / width;
-        const xOffset = borderOffset * xScale;
-        minX -= xOffset; // min-x
```

Comment:
@monfera I combined line 47 and 48. I realize I shouldn't actually be subtracting `minX` and `minY` since I already have the width and height of the SVG.